src/highlighters.cc

```diff
@@ -1130,52 +1130,56 @@ constexpr StringView LineNumbersHighlighter::ms_id;
 void show_matching_char(HighlightContext context, DisplayBuffer& display_buffer, BufferRange)
 {
     const Face face = context.context.faces()["MatchingChar"];
-    using CodepointPair = std::pair<Codepoint, Codepoint>;
-    static const CodepointPair matching_chars[] = { { '(', ')' }, { '{', '}' }, { '[', ']' }, { '<', '>' } };
+    // static const CodepointPair matching_chars[] = { { '(', ')' }, { '{', '}' }, { '[', ']' }, { '<', '>' } };
```

Comment:
You should remove this line, rather than just commenting it out.

Reply:
Oh that's embarrassing.